### DISCUSSION TITLE
Why these colours?

### DISCUSSION THREAD

**Discussion by TreyWW** [saadeghi/daisyui#2609] [General]
Hey there,
I've loved daisy UI since the minute i found it, and i love the new primary colour, but i really don't understand where secondary and accent fit in.

![image](https://github.com/saadeghi/daisyui/assets/73353716/56af3479-bcf4-420b-b2fa-0a6f8457ed9a)

I've not yet found a single use case / design that works with secondary let alone accent - i've just been using primary & white everywhere.

Just wondering if there's a particular reason for the colours to be like that or any examples of where it looks good? I just think the bright pink is a bit of an odd contrast to the soft base and soft primary.
I get it that I could use a custom theme, and just change secondary & accent but I don't know what to and I'm more just wondering why it's actually like how it is.

Thanks, it's an amazing library none the less

**Comment by saadeghi** [comment]
You're right, most UIs are just enough with 2 tones and primary is enough.  
However there are many designs that need a second and a third color. It's not mandatory to use them all but it's important to cover the needs of the community.  
daisyUI color system is aimed to cover the requirement for implementing all possible UIs a designer would provide so we have to include the color names for a second and a third brand color.

Let me know if you have any questions.

**Reply by TreyWW** [reply]
Ah I see, i mean i do see he need for a second/third colour, nearly all my use cases i've wanted one. It's just secondary and accent are very opposite to primary. Like primary is quite smooth and a lovely colour, but secondary is just bright and looks really odd compared to the others.

Just wondered if that's normal, or maybe it needs adjusting? I love the new primary colour since a few versions ago though, it's just nice

**Reply by saadeghi** [reply]
The brightness of secondary color depends on the theme (like every other color)  
And the default themes only exist to speedup the development.  
I always encourage people to make a custom theme for their project because that's what makes the website unique to the brand.

**Reply by TreyWW** [reply]
Ah okay, yeah I'll definitely use custom colours then. Yeah thanks for that, it's an amazing product. Thanks for letting me know, really appreciate it
